src/swagger/swagger.ts

```diff
@@ -318,11 +318,11 @@ export namespace Swagger {
     flow: 'accessCode';
     tokenUrl: string;
     authorizationUrl: string;
-    scopes?: OAuthScope[];
```

Comment:
if this change doesn't break any tests.. does that mean we're not testing this code yet?